src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -26,12 +26,24 @@ export const AppContainer = ({ view, params }: { view?: View; params: AppMountPa
       {/* TODO: improve fallback state */}
       <Suspense fallback={<div>Loading...</div>}>
         <Context {...params}>
-          <Sidebar>
-            <Panel {...params} />
-          </Sidebar>
-          <EuiPageBody className="deLayout__canvas">
-            <Canvas {...params} />
-          </EuiPageBody>
+          <EuiResizableContainer>
+            {(EuiResizablePanel, EuiResizableButton) => (
+              <>
+                <EuiResizablePanel initialSize={140} minSize="10%" mode="collapsible">
```

Comment:
```suggestion
                <EuiResizablePanel initialSize={140} minSize="10%" mode="collapsible" paddingSize="none">
```

Reply:
Nit: would be nice to give the left panel a bit right padding after setting `paddingSize="none"` , like `style={{ paddingRight: 8 }}`, otherwise the resize button is overlapping with the right border:
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/486382/1fb48a67-649e-4a23-8b55-26fd15e65ebe)

Reply:
Thats fine for now. I'd leave minor modifications to spacing and padding to a full UX review since they have to match other experiences too.

Reply:
Sounds good, I'm fine with that :)

Reply:
Updated `style={{ paddingRight: 8 }}`